sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml

```diff
@@ -87,7 +87,7 @@ extensions:
       timeout: 10800
       script: |-
         for i in $(docker images --format '{{.Repository}}:{{.Tag}}' | grep "openshift\/origin"); do
-          sudo skopeo copy docker-daemon:$i containers-storage:$i
+          sudo skopeo copy docker-daemon:$i containers-storage:[overlay2@/var/lib/containers/storage+/var/run/containers/storage:overlay2.override_kernel_check=1]$i
```

Comment:
do I have to quote brackets here? cause I have to in my shell but I'm not sure about it here @kargakis @stevekuznetsov

Reply:
Yeah, you probably need to escape these. Not 100% sure though but we can try it.

Reply:
so I'll repush with those escaped? or we can try this commit first?

Reply:
alright I've re-pushed with brackets escaped